src/nrtk_explorer/app/transforms.py

```diff
@@ -216,10 +212,10 @@ def _update_images(self, selected_ids):
             img = self.context.images_manager.load_image(image_filename)
 
             self.state[image_id] = images_manager.convert_to_base64(img)
-            self.state[meta_id] = {
-                "width": image_metadata["width"],
-                "height": image_metadata["height"],
-            }
+            self.update_image_meta(
```

Comment:
Why is this change needed?

Reply:
To feed a grab-bag of info to the client associated with dataset image IDs.  The transform image distance is computed elsewhere and merged with the "meta" dict.

Reply:
aha! I see, we need this to keep adding items to the dictionary corresponding of the meta attributes of an image. That is why the TypeDict in the image_meta.py which lets you append items (upto what it is allowed).